swarm-test/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.1.0 - unreleased
+
+- Initial release.
```

Comment:
```suggestion
- Initial release.

```

Reply:
Why an additional newline? Note that there is already one newline character at the end of the final line, thus GitHub is not complaining.

Reply:
![Screenshot_20230320-122913.png](https://user-images.githubusercontent.com/5486389/226326876-b33e1f6f-fbd0-4d7e-83ac-bd8c717dbe78.png)

Hmm, it is for me?

Reply:
Strange. Neither on Firefox nor Chromium it shows for me. Though I was wrong, there is no newline. Will push a commit.